plugin/trino-hive/src/main/java/io/trino/plugin/hive/authentication/HdfsKerberosConfig.java

```diff
@@ -55,4 +57,18 @@ public HdfsKerberosConfig setHdfsTrinoKeytab(String hdfsTrinoKeytab)
         this.hdfsTrinoKeytab = hdfsTrinoKeytab;
         return this;
     }
+
+    @NotNull
+    public Optional<@FileExists String> getHdfsTrinoCredentialCacheLocation()
+    {
+        return Optional.ofNullable(hdfsTrinoCredentialCacheLocation);
+    }
+
+    @Config("hive.hdfs.trino.credential-cache")
+    @ConfigDescription("Trino credential-cache location used to access HDFS")
```

Comment:
Can we use keytab together with cache? Do we need validation for this?

Reply:
Using a keytab along with cache is also a valid configuration. So I don't think we need a verification for it. Maybe we can add an additional config to check if either of them are present.

Reply:
Can we have a test for this then?

Reply:
I think we do have for them.

Reply:
Additional PT environment for both keytab with cache ?

Reply:
Yes. WDYT?

Reply:
We can add but I'm worried about 4 additional environments and if it would increase the CI runtime.

Reply:
How about having single environment with multiple catalogs?

Reply:
then we need to multiply the tests. Instead of copying tests I think adding more environment would give us better coverage.

Reply:
You can make tests parametric

Reply:
It is also kind of tricky - if we are testing for group `configured_features` then we need to make all the tests parametric but some of them would run in a non-kerberos environment. So we need to duplicate them for kerberos and non-kerberos environment - or introduce some dummy catalogs to non-keberos environment.

Reply:
OK. So let's add new environment then.

Reply:
Added